### DISCUSSION TITLE
Windows 7 no longer supported?

### DISCUSSION THREAD

**Discussion by BlohoJo** [AdguardTeam/AdGuardHome#6803] [Q&A] [answered]
The AdGuard Home service no longer starts on a Windows 7 machine starting with version 0.107.45. Starting the service returns the error `The service did not respond to the start or control request in a timely fashion.` This error comes up immediately.

Just checking to confirm that this is intended.

**Comment by ainar-g** [comment, marked as answer]
Yes.  As is written in the changelog and the release notes, previously deprecated support for Go 1.20 (the last Go release to support Windows versions below 10) has been removed.  See [Go 1.21 release notes](https://go.dev/doc/go1.21#windows) as well.  Microsoft ended support for Windows 7 about four years ago, if I recall correctly, so we recommend updating to a newer OS if possible.

**Comment by BlohoJo** [comment]
Understood. :disappointed:

**Comment by IRainman** [comment]
That's sad because Windows Server 2008R2 is still supported.
